src/test/java/org/jqassistant/plugin/jee/cdi/test/set/beans/scope/jakarta/JakartaTypeWithConversationScopedField.java

```diff
@@ -0,0 +1,12 @@
+package org.jqassistant.plugin.jee.cdi.test.set.beans.scope.jakarta;
+
+import jakarta.enterprise.context.ConversationScoped;
+
+import org.jqassistant.plugin.jee.cdi.test.set.beans.scope.ProducedBean;
+
+public class JakartaTypeWithConversationScopedField {
+
+    @ConversationScoped
+    ProducedBean producedBean;
```

Comment:
Please verify if this is supported by CDI or if Produces is required in addition. If not then this class and according rules shall be removed.